dataland-frontend/src/utils/Constants.ts

```diff
@@ -30,4 +30,7 @@ export const TIME_DELAY_BETWEEN_SUBMIT_AND_NEXT_ACTION_IN_MS = 2000;
 
 export const UPLOAD_FILE_SIZE_DISPLAY_DECIMALS = 2;
 
+// Same as standard for windows machines. File name can't have one of the special characters <>:"|?/\ and shouldn't end with a whitespace or .
+export const REGEX_FOR_FILE_NAMES = /^(?=.{1,254}$)[^<>:"|?/*\\]+[^<>:"|?/*\\.\s]$/;
```

Comment:
This should be used in the backend as well

Reply:
From a consistency point of view: Yes.
But I think it won't break anything if we don't have it in the backend.

Or do you see potential problems that could occur?

Reply:
Ah ok yes that would indeed be a problem.  Good point.  WIll include it in the backend